spec/feature_spec.rb

```diff
@@ -0,0 +1,3 @@
+require './lib/order.rb'
+require './lib/menu.rb'
+require './lib/sms.rb'
```

Comment:
Were you planning to build this out further? I need to get into the habit of building feature_specs.